EpsilonCCS/Inc/Data/MpptData.h

```diff
@@ -9,11 +9,10 @@
 struct MpptData
 {
     unsigned char alive;
-    unsigned char mpptNumber;
     unsigned short int arrayVoltage;
     unsigned short int arrayCurrent;
     unsigned short int batteryVoltage;
     unsigned short int temperature;
 };
 
-extern struct MpptData mpptData;
```

Comment:
Maybe it might be good to expose another const of the length of this array (how many mppts). Things using this class probably wouldn't know otherwise.